src/logic/waila.ts

```diff
@@ -1,10 +1,7 @@
-import initWaila, { PaymentParams } from "@mutinywallet/waila-wasm";
+import { PaymentParams } from "@mutinywallet/mutiny-wasm";
 
 import { Result } from "~/utils";
 
-// Make sure we've initialzied waila before we try to use it
-await initWaila();
```

Comment:
Is there any concern with mutiny-wasm not being initialized yet?

Reply:
no, if we try to use waila before mutiny-wasm is initialized we have bigger problems